homeassistant/components/zwave_js/manifest.json

```diff
@@ -30,5 +30,8 @@
     }
   ],
   "zeroconf": ["_zwave-js-server._tcp.local."],
-  "loggers": ["zwave_js_server"]
+  "loggers": ["zwave_js_server"],
+  "supported_brands": {
+    "leviton": "Leviton"
```

Comment:
How will users differentiate between the different Leviton integrations?

Reply:
The BlueTooth version will be removed
The wifi version is currently YAML only

Reply:
Ok, so this won't affect our docs? Ie there won't be a docs entry for Leviton which leads to Z-Wave?

Reply:
There will be a Leviton Docs page yes

Reply:
Won't that be confusing? I think we should make sure we don't increase confusion since the aim with supported brands is to make it less confusing for users how to set up their device.

Reply:
I would rather not add more than the Brand name its self but if needed we can change it to `Leviton Z-Wave`

Reply:
Adding a unique suffix follows the idea with a landing page integration, with the brand name, and brand name + suffix for the linked integrations, if a brand has more than one integration. So that sounds good.

Reply:
```suggestion
    "leviton": "Leviton Z-Wave"
```